samples/cpp/hello_reshape_ssd/main.cpp

```diff
@@ -53,7 +53,7 @@ int main(int argc, char* argv[]) {
         // try to find it.
         ov::NodeVector ops = model->get_ops();
         auto it = std::find_if(ops.begin(), ops.end(), [](std::shared_ptr<ov::Node> node) {
-            return node->get_type_info() == ngraph::op::DetectionOutput::get_type_info_static();
+            return node->get_type_info().name == ngraph::op::DetectionOutput::get_type_info_static().name;
```

Comment:
@vladimir-dudnik It is not correct way, if you want to find some specific operation. Because you can have different operation versions in different opsets...
Maybe in this case will be better to use dynamic cast or compare DiscreteTypeInfo.

But it will work if you want to find any version of some specific operation. Is it what you want?

Reply:
I think for DetectionOutput it is enough to find any version, I do not expect much changes between versions. Is my assumption correct? The idea is that sample claim it support models with SSD output, so we just want to ensure such output exist